### ISSUE TITLE
[ST2110-21] TROFF... positive integer or non-negative integer?

### ISSUE THREAD

**Issue by garethsb** [SMPTE/st2110-10#10]
**_Note: Raising issue in this repo, since ST 2110-21 still doesn't have its own (#7)._**

![image](https://github.com/SMPTE/st2110-10/assets/31761158/c48f678e-bc79-4386-9936-5db3e0829b38)

Regarding `TROFF`, ST 2110-21 section 8.2 says "positive integer"...

I now realise _positive_ is probably incorrect here, and non-negative (positive or zero) is intended, as per section 6.2?

![image](https://github.com/SMPTE/st2110-10/assets/31761158/21a4735b-006d-4cd5-9955-8ba022755124)

**Comment by jmailhot** [comment]
agree, we should put this on a list to rectify in some future revision; however it does not (IMHO) rise to the severity that an advisory note might require.   Recommend that we close this issue, but start a "to be addressed" log for 2110.
